localstack/testing/pytest/stepfunctions/fixtures.py

```diff
@@ -427,3 +571,74 @@ def _setup(event_pattern):
         return queue_url
 
     return _setup
+
+
+@pytest.fixture
+def sfn_events_to_sqs_queue(events_to_sqs_queue):
```

Comment:
cc @maxhoheiser We now have SFN fixtures in localstack, which can be useful for integrations in EventBridge and EventBridge Pipes 🥳